modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/service/persistence/impl/DDMFormInstanceFinderImpl.java

```diff
@@ -178,6 +224,27 @@ protected int doCountByKeywords(
 			inlineSQLHelper);
 	}
 
+	protected int doCountByKeywords(
```

Comment:
@linolaoj, this method is duplicating code when compared with the existing method `doCountByKeywords` just because of the new parameter `status`. Can you please re-use the existing logic instead and do something similar too what is being done on method `addFormInstance`? The method `addFormInstance` has a single logic shared among all methods named `addFormInstance`. Please, check if it's necessary do this same change on other parts of this pull. 😉 

Thanks.